src/DataLakeStore/DataLakeStore/DataLakeStore.csproj

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DataLake.Store" Version="1.1.15" />
+    <PackageReference Include="Microsoft.Azure.DataLake.Store" Version="1.2.1-alpha" />
```

Comment:
We named it as alpha but it would be same as beta only. https://www.nuget.org/packages/Microsoft.Azure.DataLake.Store/1.2.1-alpha Do you want this to beta only? Do you want it to be beta only? We can make it beta since the difference would be the name.